validate_test.go

```diff
@@ -46,7 +46,8 @@ func TestValidateCids(t *testing.T) {
 	}{
 		{mhcid(mh.SHA2_256, 32), nil},
 		{mhcid(mh.SHA2_256, 16), ErrBelowMinimumHashLength},
-		{mhcid(mh.MURMUR3, 4), ErrPossiblyInsecureHashFunction},
+		{mhcid(mh.MURMUR3X64_64, 4), ErrPossiblyInsecureHashFunction},
+		{mhcid(mh.BLAKE3, 32), nil},
```

Comment:
Please add a test with a bigger random size like `42` or `69`.

Reply:
done, and also added a test case for rejecting a CID that magically ended up with 129 blake3 bytes